src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,5 +1,28 @@
 import React from 'react';
+import './CommentInfo.scss';
 
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
+type Props = {
+  commentName: string;
+  email: string;
+  commentBody: string;
+};
+
+export const CommentInfo: React.FC<Props> = ({
+  commentName,
+  email,
+  commentBody,
+}) => (
+  <li className="CommentInfo">
+    <div className="CommentInfo__title">
+      <strong className="CommentInfo__name">{commentName}</strong>
+
+      {' by '}
+
+      <a className="CommentInfo__email" href={`mailto:${email}`}>
```

Comment:
Add a 'rel' attribute with value 'noopener noreferrer' to the 'a' element for better security.